source/MaterialXContrib/MaterialXNode/CreateMaterialXNodeCmd.cpp

```diff
@@ -47,46 +46,62 @@ MStatus CreateMaterialXNodeCmd::doIt( const MArgList &args )
 		return status;
 
 	MString materialXDocument;
-	MString elementName;
+	MString elementPath;
 	if( parser.isFlagSet(kDocumentFlag) )
 	{
 		argData.getFlagArgument(kDocumentFlag, 0, materialXDocument);
 	}
 	if (parser.isFlagSet(kElementFlag))
 	{
-		argData.getFlagArgument(kElementFlag, 0, elementName);
+		argData.getFlagArgument(kElementFlag, 0, elementPath);
 	}
-	if (materialXDocument.length() > 0 && elementName.length() > 0)
+	if (materialXDocument.length() > 0 && elementPath.length() > 0)
 	{
-		// Load document
-		MaterialX::DocumentPtr doc = MaterialX::createDocument();
-		MaterialX::readFromXmlFile(doc, materialXDocument.asChar());
+        std::unique_ptr<MaterialXData> materialXData{
```

Comment:
Seems to be indentation issues. Please set indentation in your text editor to insert spaces instead of tabs. Our convention is to use 4 spaces. Cheers

Reply:
That's the way I'm set up but previous versions of the file were using tabs. FYI @bernardkwok @feldstj :-)